pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -203,35 +262,44 @@ func RunStatus(cmd *cobra.Command, args []string) {
 		log.Fatalf(`❌ Missing expected arguments: <noobaa-account-name> %s`, cmd.UsageString())
 	}
 
+	name := args[0]
+
 	o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaaaccount_cr_yaml)
 	noobaaAccount := o.(*nbv1.NooBaaAccount)
 	secret := util.KubeObject(bundle.File_deploy_internal_secret_empty_yaml).(*corev1.Secret)
 
-	noobaaAccount.Name = args[0]
-	secret.Name = fmt.Sprintf("noobaa-account-%s", args[0])
+	noobaaAccount.Name = name
+	secret.Name = fmt.Sprintf("noobaa-account-%s", name)
 	noobaaAccount.Namespace = options.Namespace
 	secret.Namespace = options.Namespace
 
-	if !util.KubeCheck(noobaaAccount) {
+	if !util.KubeCheck(noobaaAccount) && (name != "admin@noobaa.io") {
```

Comment:
I don't think we need to change the status - this is for CRD only accounts

Reply:
The status prints the keys from the secret and we would like to print it also for the admin account.
In the admin account, the secret name is different from the email chance we need to change the name to and get the proper secret.
also in the admin account, there is no CRD so we don't want to check it.

Reply:
I have to say that I don't this special handling - but after I understand that's it's a must - go for it.